config/config_test.go

```diff
@@ -0,0 +1,210 @@
+package config_test
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"testing"
+
+	"github.com/ONSdigital/log.go/v3/config"
+	"github.com/ONSdigital/log.go/v3/log"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestFromEnv(t *testing.T) {
+	Convey("Starting with a clear environment", t, func() {
+		os.Clearenv()
+
+		Convey("No vars leaves config unchanged", func() {
+			wanted := config.Config{Namespace: "some namespace"}
+			cfg := wanted
+			config.FromEnv(&cfg)
+			So(cfg, ShouldEqual, wanted)
+		})
+
+		Convey("Setting HUMAN_LOG updates pretty config", func() {
+			cfg := config.Config{}
+
+			tests := []struct {
+				value  string
+				wanted bool
+			}{
+				{"1", true},
+				{"0", false},
+				{"t", true},
+				{"f", false},
+				{"true", true},
+				{"false", false},
+			}
+
+			for _, test := range tests {
+				Convey("When HUMAN_LOG="+test.value, func() {
+					os.Setenv("HUMAN_LOG", test.value)
```

Comment:
There's an unhandled error here.
```suggestion
					err := os.Setenv("HUMAN_LOG", test.value)
					So(err, ShouldBeNil)
```

Reply:
Thank you, addressed in https://github.com/ONSdigital/log.go/pull/44/commits/fc70a349237c838e973a6e61addef0ee83910665